### ISSUE TITLE
Allow users to add custom buff Names or IDs to current spell lists

### ISSUE THREAD

**Issue by TheInternetIsReal** [jordonwow/bigdebuffs#167]
Hi,

I noticed there was no way to add custom spells to any of the spell lists. I was going to open a ticket asking you to add Shadowy Duel to "Other Buffs" and Guardian of the Ancient Queen under "Immunities" (it's incorrectly under defensive buffs atm) but decided on this instead.

Is this possible in any way today? Maybe by editing the lua files?

**Comment by Kennah229** [comment]
Go into

C:\Program Files (x86)\World of Warcraft_retail_\Interface\AddOns\BigDebuffs\Retail

and add (for example)

[110909] = { type = BUFF_DEFENSIVE }, -- Alter Time

close and save. the first numbers are the spell ID found on wowhead. change spell ID/type/name for each spell you want to add.
to save myself time i honestly just throw everything under "BUFF_OTHER" all the way at the bottom. here are the spells I have added. 


    [87023] = { type = BUFF_OTHER }, -- Cauterize
    [209426] = { type = BUFF_OTHER }, -- Darkness
    [45182] = { type = BUFF_OTHER }, -- Cheating Death
    [345231] = { type = BUFF_OTHER }, -- Gladiator's Emblem
    [317589] = { type = BUFF_OTHER }, -- Tormenting Backlash
    [11327] = { type = BUFF_OTHER }, -- Vanish
    [25771] = { type = BUFF_OTHER }, -- Forbearance
    [125174] = { type = BUFF_OTHER }, -- Touch of Karma
    [116095] = { type = BUFF_OTHER }, -- Disable
    [2983] = { type = BUFF_OTHER }, -- Sprint
    [320224] = { type = BUFF_OTHER }, -- Podtender
    [331866] = { type = BUFF_OTHER }, -- Agent of Chaos
    [310143] = { type = BUFF_OTHER }, -- Soulshape
    [212792] = { type = BUFF_OTHER }, -- Cone of Cold
    [110909] = { type = BUFF_OTHER }, -- Alter Time
    [34709] = { type = BUFF_OTHER }, -- Shadow Sight
    [185422] = { type = BUFF_OTHER }, -- Shadow Dance

**Comment by TheInternetIsReal** [comment]
Thanks, I'll try that for now.

**Comment by ManneN1** [comment]
This is an OK workaround, the problem is that whatever manual changes you make to the lua files are overriden whenever Jordon updates the AddOn and you download the latest version.

Adding in a similar function as in OmniBar would be optimal.

**Comment by Kennah229** [comment]
You're right. But honestly BigDebuffs isn't updated all that frequently. I just made a backup of the Retail file with all my customs spells and whenever it is updated, I'll throw the custom spells back in.
